docs/spec/bento-common-props.md

```diff
@@ -0,0 +1,27 @@
+# Common Props in Preact/React Bento Components
+
+The following are common props that should be be supported in al Bento Preact/React Bento Components when possible.
```

Comment:
```suggestion
The following are common props that are supported in all Bento Preact/React Bento Components.
```

Reply:
I'm fine with taking your phrasing, but I worded it like that because these props aren't actually supported in all components. Do we/should we consider it a bug if a component doesn't support all these props?

Reply:
Yes, let's file an issue to keep track of this effort. Most components should support these by default via `...rest` props, but if they are destructured out for whatever reason we should just make sure they continue to be propagated.